micro-slack/build.gradle

```diff
@@ -0,0 +1,11 @@
+description = 'micro-slack'
+
+dependencies {
+    compile 'com.aol.microservices:micro-grizzly-with-jersey:0.87'
+    compile 'com.aol.microservices:micro-general-exception-mapper:0.87'
+    
+    testCompile group: 'junit', name: 'junit', version:'4.10'
+    testCompile  'org.springframework.boot:spring-boot-starter-test:1.1.5.RELEASE'
```

Comment:
There is a properties file that can be edited for the webhook url in case anyone wants to test it using a different one. Let me know if you want this option removed.